src/create-request.js

```diff
@@ -49,6 +49,7 @@ function createRequestWithMethod (type, definition, method) {
     }
   }
   actionCreator.toString = () => LP_API_ACTION_NAMESPACE + type
```

Comment:
Should you deprecate `toString()` in favor of `type`?

Reply:
Eh, that would be a breaking change. I wasn't super convinced by the rationale to remove it from redux-toolkit – the issue itself was pretty sparse

Reply:
I was thinking more about a warning if you used it, not complete removal. But that makes sense!